pkg/ddl/ddl_api.go

```diff
@@ -6898,6 +6924,9 @@ func (d *ddl) renameTables(ctx sessionctx.Context, oldIdents, newIdents []ast.Id
 		BinlogInfo: &model.HistoryInfo{},
 		Args:       []interface{}{oldSchemaIDs, newSchemaIDs, tableNames, tableIDs, oldSchemaNames, oldTableNames},
 		CtxVars:    []interface{}{append(oldSchemaIDs, newSchemaIDs...), tableIDs},
+
+		AffectedSchemaNames: affectedSchemaNames,
+		AffectedTableNames:  affectedTableNames,
```

Comment:
Could we put these fields to `MultiSchemaInfo`?
There seems to be some overlap between `TableName` and `AffectedTableNames`.
But in that case, the `rename table` operation is a little tricky to explain. Or we can specially handle this operation by getting the `RawArgs` information.

Reply:
No, 'MultiSchemaInfo' is only used for multi-schema changes, and this PR is not related to multi-schema changes.

`rename table` may change `RawArgs` during execution, with increasing complexity.

Reply:
Or consider putting some related fields into a field of this Job.